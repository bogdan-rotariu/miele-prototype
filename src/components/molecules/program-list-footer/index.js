export { ProgramListFooter } from './program-list-footer'
