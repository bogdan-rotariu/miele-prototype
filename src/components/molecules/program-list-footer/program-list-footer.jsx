import { useState } from 'react'
import { Button } from '../../atoms/styled-button'
import styles from './program-list-footer.module.scss'

export const ProgramListFooter = () => {
    const [isActiveBtn, setIsActiveBtn] = useState(false)

    return (
        <footer className={styles.footer}>
            <Button
                onClick={() => setIsActiveBtn(!isActiveBtn)}
                isActive={isActiveBtn}
            >
                Back
            </Button>
            <Button isDisabled>Inactive Button</Button>
        </footer>
    )
}
