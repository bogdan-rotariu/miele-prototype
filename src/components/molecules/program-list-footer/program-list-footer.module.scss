.footer {
    display: flex;
    justify-content: space-between;
    padding: 0.375rem;
}
