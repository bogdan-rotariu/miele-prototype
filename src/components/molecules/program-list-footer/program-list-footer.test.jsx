import { render } from '@testing-library/react'
import { ProgramListFooter } from './program-list-footer'

describe('ProgramListFooter', () => {
    it('should match snapshot', () => {
        const { container } = render(<ProgramListFooter />)
        expect(container.firstChild).toMatchSnapshot()
    })
})
