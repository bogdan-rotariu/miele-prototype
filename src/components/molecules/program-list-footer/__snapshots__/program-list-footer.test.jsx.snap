// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProgramListFooter should match snapshot 1`] = `
<footer
  class="footer"
>
  <button
    class="button"
  >
    Back
  </button>
  <button
    class="button disabled"
    disabled=""
  >
    Inactive Button
  </button>
</footer>
`;
